docs/pipelines/fault-tolerance.md

```diff
@@ -178,13 +205,11 @@ request writing a checkpoint immediately.  If automatic checkpoints
 are disabled, then the user should occasionally invoke this API to
 ensure that the checkpoint feature is useful.
 
-<!-- Uncomment this when Delta Lake supports checkpointing (see
-https://github.com/feldera/feldera/issues/3884):
 > ⚠️ Writing a checkpoint should ordinarily be a fairly fast operation
 that takes a second or less.  However, the [Delta Lake input
```

Comment:
```suggestion
that takes several seconds.  However, the [Delta Lake input
```

even for small pipelines, I see checkpoint times in the order of 10s. I don't think I've ever seen it complete in <1s.